### ISSUE TITLE
docx - pdf conversion problem: Line Spacing and Character spacing greater in pdf

### ISSUE THREAD

**Issue by karbert** [opensagres/xdocreport#246]
Conversion with PdfConverter:
----------------------------------

import org.apache.poi.xwpf.usermodel.XWPFDocument;
import fr.opensagres.poi.xwpf.converter.pdf.PdfConverter;
import fr.opensagres.poi.xwpf.converter.pdf.PdfOptions;
...
XWPFDocument document = null;
try {
	document = new XWPFDocument(is);
	final PdfOptions pdfOptions = null;
	PdfConverter.getInstance().convert(document, os, pdfOptions);
} catch (final Exception e) {
	...
} finally {
	document = null;
}

fr.opensagres.xdocreport:             last version: 2.0.1
org.apache.poi:                             last version: 3.17

Conversion success, but Line Spacing and Character spacing greater in pdf.
From 1,5 page long docx convert 2,5 page long pdf.

**Comment by angelozerr** [comment]
Any contribution are welcome!

**Comment by karbert** [comment]
My small testdata: only 3 lines, with Times New Roman 10

input docx file: 
[small.docx](https://github.com/opensagres/xdocreport/files/1359150/small.docx)

perfect pdf created with Acrobat/Create pdf command in MS Word: 
[small_with_acrobat.pdf](https://github.com/opensagres/xdocreport/files/1359156/small_with_acrobat.pdf)

POI converted file:
[small_with_POI.pdf](https://github.com/opensagres/xdocreport/files/1359152/small_with_POI.pdf)

view lines in Adobe Acrobat Pro in Edit mode:
![small_with_acrobat_lines](https://user-images.githubusercontent.com/32540672/31225813-1b903b3e-a9d4-11e7-86c5-8acc4e9e23b4.jpg)
![small_with_poi_lines](https://user-images.githubusercontent.com/32540672/31225819-1e361886-a9d4-11e7-8797-275949a41a5b.jpg)

In this 3 lines can see difference.

**Comment by ballgithub** [comment]
@imajic
may I know where could I get your fix

**Comment by SMesser** [comment]
Clicking on the pull request and then on the "Files changed" tab takes you to @imagic 's  patch: https://github.com/opensagres/xdocreport/pull/250/files

**Comment by SMesser-WhiteHawk** [comment]
Since @imajic hasn't kept his PR up-to-date and merging was being blocked by merge conflicts, I forked base and merged in imajic's changes, resolved the merge conflicts, and opened a new PR here: https://github.com/opensagres/xdocreport/pull/365

**Comment by Schlonz** [comment]
it's 2022. Any chances that this pull request will be merged in the future?

**Comment by angelozerr** [comment]
If someone resolve conflict, I'm opened to merge it.

**Comment by ase379** [comment]
@angelozerr There are no conflicts and in every version i am picking this snippet to avoid a formatting mess. Can you approve and merge it ?

**Comment by angelozerr** [comment]
@ase379 if you speak about https://github.com/opensagres/xdocreport/pull/365 github said me that there is conflicts.

If you want to create a new PR, I will be happy to merge it. Thanks.

**Comment by ase379** [comment]
That is strange, in my pr view it is marked as "no conflicts".

Can you state the Problem ?

**Comment by angelozerr** [comment]
I see that in github:

![image](https://github.com/opensagres/xdocreport/assets/1932211/f710e7ed-15ae-4036-b805-6534884511d5)

I have no time to investigate the problem, please create a new PR.

**Comment by ase379** [comment]
@angelozerr I made a new PR from a private fork.. can you please check it ?
